### DISCUSSION TITLE
Image cache busting (Pimcore X)

### DISCUSSION THREAD

**Discussion by valentin-** [pimcore/pimcore#10559] [Q&A / Support] [answered]
Hi, I'd like to activate cache busting for images in twig templates. As the source can vary inside, images get rendered using `pimcore_image()` or `image.thumbnail()`.

There is a cache busting option for pimcore_image():

![image](https://user-images.githubusercontent.com/90033/136910732-a9d57a8c-0e3e-453a-9472-e006d890097a.png)

But it looks like cache busting can't get activated for image.thumbnail:

https://pimcore.com/docs/pimcore/current/Development_Documentation/Assets/Working_with_Thumbnails/Image_Thumbnails.html#page_Generating-HTML-for-Thumbnails

Is this somehow possible?

Thanks!

**Comment by brusch** [comment, marked as answer]
```twig
{{ image.thumbnail("foo").html({ cacheBuster: true }) }}
```
